lib/features/auth/view/widget/circular_button.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+import 'package:telware_cross_platform/core/theme/palette.dart';
+
+class CircularButton extends StatelessWidget {
+  const CircularButton({
+    super.key,
+    required this.icon,
+    required this.iconSize,
+    this.paddingTop = 0,
+    this.paddingBottom = 0,
+    this.paddingLeft = 0,
+    this.paddingRight = 0,
+    required this.radius,
```

Comment:
what is the point of taking the 4 padding directions if you are not going to use them all in the Padding widget?

Reply:
This class is made for general use not just for my case, so I added them all.